plugins/amd-kria/fu-amd-kria-device.c

```diff
@@ -126,20 +126,17 @@ fu_amd_kria_device_probe(FuDevice *device, GError **error)
 	 * Fetch the OF_FULLNAME udev property and look for the I2C address in it
 	 * sample format: OF_FULLNAME=/axi/i2c@ff030000/eeprom@50
 	 */
-	tmp = g_udev_device_get_property(udev_device, "OF_FULLNAME");
-	if (tmp == NULL) {
-		g_set_error_literal(error,
-				    FWUPD_ERROR,
-				    FWUPD_ERROR_NOT_SUPPORTED,
-				    "no OF_FULLNAME property");
+	prop_of_fullname =
+	    fu_udev_device_read_property(FU_UDEV_DEVICE(device), "OF_FULLNAME", error);
+	if (prop_of_fullname == NULL)
```

Comment:
I still think it would be better to use this as physical ID (move to libfwupdplugin) and abstract this from the plugin.   But if you'd rather that be a follow up that's fine.

Reply:
Yes, that's exactly what I'm moving towards -- I want to drop the set_logical_id() and set_physical_id() completely from each plugin and just do it automatically in the baseclass.